### PR TITLE
Update edit.html.erb

### PR DESCRIPTION
**Pull request by Kateharries**
### Changes proposed in this pull request

Body copy paragraph so that providers have a bit more information on visas.

### Checklist

- [ ] Publish / TTAPI Merge - does this code change affect a part of the app that's currently being migrated to TTAPI? If so, speak with the dev doing the migration and ensure they've accounted for this change.
- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [ ] Rebased master
- [ ] Cleaned commit history
- [ ] Tested by running locally
- [ ] Product Review

### CODE REVIEW

File: app/views/providers/visas/edit.html.erb [hunk 1]
```diff
@@ -22,9 +22,10 @@
         <%= page_title %>
       </h1>
 
-      <p class="govuk-body">If you’re unsure if your courses can sponsor visas, check the <%= govuk_link_to("Register of Student sponsors", "https://www.gov.uk/government/publications/register-of-licensed-sponsors-students") %> and <%= govuk_link_to("Register of Worker and Temporary Worker licensed sponsors", "https://www.gov.uk/government/publications/register-of-licensed-sponsors-workers") %>.</p>
+      <p class="govuk-body">Non-UK teacher training candidates need the correct visa or immigration status to work or study in England.</p>
+    <p class="govuk-body">Student visas are sponsored by the teacher training provider that ratifies your course. Skilled Worker visas are sponsored by the employing school, local authority or multi-academy trust. </p>
```
Comment (defong) [inline app/views/providers/visas/edit.html.erb:26]:
```suggestion
      <p class="govuk-body">Student visas are sponsored by the teacher training provider that ratifies your course. Skilled Worker visas are sponsored by the employing school, local authority or multi-academy trust.</p>
```

**Comment by defong** [comment]
I think this is good to go, but may need to be ported to ttapi, double checking with @darokel

**Review by Kateharries — COMMENTED** [review]
There was a rogue full stop after the link so I've just deleted it.

**Review by defong — APPROVED** [review]
@darokel port is underway

**Review by mbocc — COMMENTED** [review]
Great content changes - all approved, thank you!

**Review by mbocc — APPROVED** [review]
approved!

**Pull request merged**